COPY/etc/httpd/conf.d/manageiq-redirects-websocket

```diff
@@ -1,2 +1,2 @@
-ProxyPass /ws balancer://evmcluster_websocket/ws
-ProxyPassReverse /ws balancer://evmcluster_websocket/ws
+ProxyPass /ws/console balancer://evmcluster_websocket/ws/console
+ProxyPassReverse /ws/console balancer://evmcluster_websocket/ws/console
```

Comment:
`/ws/console` traffic will go to the console.
This is the same as before.

The difference being that console traffic is not traveling along a more specific url.
And this is now avoiding the rest of the `/ws/` url namespace